README.md

```diff
@@ -75,7 +75,7 @@ Kirjastoriippuvuuksien haavoittuvuudet voi tarkastaa joko juuritasolta koko proj
 ### oikeustulkkirekisteri-service
 
 Bäkkäriä voi ajaa suorittamalla `oikeustulkkirekisteri-service` -hakemistossa:
-`mvn cargo:run`
+`mvn package cargo:run`
```

Comment:
En saa bäkkäriä lokaalisti pyörimään:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project oikeustulkkirekisteri-service: Fatal error compiling: java.lang.IllegalAccessError: class lombok.javac.apt.LombokProcessor (in unnamed module @0x66075347) cannot access class com.sun.tools.javac.processing.JavacProcessingEnvironment (in module jdk.compiler) because module jdk.compiler does not export com.sun.tools.javac.processing to unnamed module @0x66075347 -> [Help 1
```